### PR TITLE
upgrade tflite

### PR DESCRIPTION
**Pull request by roywei**
- TF 2.3.2 and 2.4.0 are both released. Upgrading tflite to 2.4.0
- Add build flag for more operators
- minor fixes

https://github.com/tensorflow/tensorflow/tags

### CODE REVIEW

**Review by frankfliu — APPROVED** [review]

**Comment by roywei** [comment]
Please wait for jni updates before merge

**Pull request merged**
